### PR TITLE
Issue 5587: BookKeeper Docker Image build failure.

### PR DESCRIPTION
**Pull request by AndreyKoltsov1997**
**Change log description**  
 [BookKeeper Dockerfile](https://github.com/pravega/pravega/blob/master/docker/bookkeeper/Dockerfile) uses  `apache/bookkeeper:4.11.1` as a base image. The Dockerfile downloads [pip bootstrap script](https://github.com/pravega/pravega/blob/master/docker/bookkeeper/Dockerfile#L40) whose installation is failing since it uses newer, Python 3 syntax. Thus, BookKeeper image build fails as well.
`pip` is installed in order to install `zk-shell` into the image, however, `zk-shell` is already available within base image `apache/bookkeeper:4.11.1`, thus both pip and zk-shell installations could be removed from the Dockerfile.

**Purpose of the change**  
Fixes https://github.com/pravega/pravega/issues/5587

**What the code does** 
Removes `pip` and `zk-shell` references from [BookKeeper Dockerfile](https://github.com/pravega/pravega/blob/master/docker/bookkeeper/Dockerfile).

**How to verify it**  
Build BookKeeper docker image.

Signed-off-by: Andrey Koltsov <a99920641@gmail.com>

### CODE REVIEW

File: docker/bookkeeper/Dockerfile [hunk 1]
```diff
@@ -36,7 +36,7 @@ RUN set -x \
     && cp -r bookkeeper-all-${BK_VERSION}/* /opt/bookkeeper/ \
     && rm -rf "bookkeeper-all-${BK_VERSION}" "$DISTRO_NAME.tar.gz" "$DISTRO_NAME.tar.gz.asc" "$DISTRO_NAME.tar.gz.sha512" \
     # install zookeeper shell
-    && wget -q https://bootstrap.pypa.io/get-pip.py \
+    && wget -q https://bootstrap.pypa.io/2.7/get-pip.py \
```
Comment (RaulGracia) [inline docker/bookkeeper/Dockerfile:39]: One question: This image is based on `apache/bookkeeper:4.11.1`, which already has installed `zk-shell`, right? So my question is, can we get rid of install pip at all? `zk-shell` should be already available, if I'm not wrong.

**Review by RaulGracia — CHANGES_REQUESTED** [review]

File: docker/bookkeeper/Dockerfile (hunk 1 above)
Comment (AndreyKoltsov1997) [reply docker/bookkeeper/Dockerfile:39]: Great catch! You're right - base image `apache/bookkeeper:4.11.1` has zk-shell installed out of the box, thus I think it'd be safe to remove zk-shell from the Dockerfile.
I haven't found any `pip` usage apart from `zk-shell` installation, thus I think `pip` references could be removed from Dockerfile as well.
```
$ docker run -it --entrypoint /bin/bash apache/bookkeeper:4.11.1
...
4.11.1: Pulling from docker.io/apache/bookkeeper
...
Status: Downloaded newer image for docker.io/apache/bookkeeper:4.11.1
[root@e544aa0b1b8c bookkeeper]# zk-shell
Welcome to zk-shell (1.3.2)
(DISCONNECTED) />
```

**Comment by RaulGracia** [comment]
Thanks @AndreyKoltsov1997, the change looks good. Let's execute a branch builder on this branch so we can confirm that everything is ok before merging.

**Review by RaulGracia — APPROVED** [review]
Branch builder 745 is successful with this change.

**Comment by RaulGracia** [comment]
@AndreyKoltsov1997 please, can you change the PR description to match the current approach of the PR?

**Comment by AndreyKoltsov1997** [comment]
> @AndreyKoltsov1997 please, can you change the PR description to match the current approach of the PR?

Sure. I've just updated the PR description.

**Review by medvedevigorek — APPROVED** [review]

**Pull request merged**